Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/NetworkDAO.java

```diff
@@ -294,6 +294,24 @@ public String addNetworkStatement(String rowid, String networkId, String id, Str
             throw new QuadrigaStorageException(e);
         }
     }
+    
+    /**
+     * This would give the list of {@link INetwork} that are approved 
+     */
+    @Override
+    public List<NetworksDTO> getApprovedNetworkList() throws QuadrigaStorageException {
+        try {
+            Query query = sessionFactory.getCurrentSession().getNamedQuery("NetworksDTO.findApprovedNetworks");
+
+            @SuppressWarnings("unchecked")
+            List<NetworksDTO> listNetworksDTO = query.list();
+
+            return listNetworksDTO;
+        } catch (Exception e) {
+            logger.error("Error in fetching a network status: ", e);
```

Comment:
It throws HibernateException, do you want to me to wrap it in QuadrigaStorage exception leave it for the calling method to handle?